packages/app/.eslintrc.js

```diff
@@ -3,12 +3,16 @@ module.exports = {
   parser: 'babel-eslint',
   parserOptions: {
     ecmaVersion: 2018,
-    sourceType: 'module'
+    sourceType: 'module',
+    ecmaFeatures: {
+      legacyDecorators: true
+    }
   },
   plugins: ['ember'],
   extends: ['eslint:recommended', 'plugin:ember/recommended', 'prettier'],
   env: {
-    browser: true
+    browser: true,
+    es6: true
   },
   rules: {
     'multiline-comment-style': ['error', 'starred-block']
```

Comment:
```suggestion
    'multiline-comment-style': ['error', 'starred-block'],
    'ember/no-jquery': 'error'
```
Add back root level no jquery rule